ownCloud/UI Elements/StaticTableViewRow.swift

```diff
@@ -293,11 +293,15 @@ class StaticTableViewRow : NSObject, UITextFieldDelegate {
 
 	// MARK: - Buttons
 	convenience init(buttonWithAction action: StaticTableViewRowAction?, title: String, style: StaticTableViewRowButtonStyle = StaticTableViewRowButtonStyle.proceed, identifier : String? = nil) {
+
+		let lateralInset: CGFloat = 20
+		let cornerRaidus: CGFloat = 6
```

Comment:
cornerRaidus should be replaced by cornerRadius (typo)

Reply:
Fixed